Ash.FrontEnd/src/global/style.less

```diff
@@ -0,0 +1,20 @@
+.ash-section {
+  margin: 10px;
+  border-radius: 10px;
+  background-color: #333;

```

Comment:
Create a less variable for this, please!

Reply:
Will do